components/status-message/src/StatusMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import {FC, createElement as h, ReactElement, ReactNodeArray} from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+
+import '../assets/StatusMessage.scss';
+
+export type StatusMessageProps = StandardProps & {
+  /** Summary of status */
+  status?: string,
+  /** Links to perform status related actions */
+  actions?: ReactElement<any> | ReactNodeArray
+};
+
+export const StatusMessage: FC<StatusMessageProps> = ({
+    children,
+    classBlock,
+    classModifiers,
+    className,
+    status,
+    actions,
+    ...attrs
+  }) => {
+  const classes = classBuilder('hods-status-message', classBlock, classModifiers, className);
+
+  return status || children ? (
+    <div {...attrs} className={classes()}>
+      <p className="govuk-!-font-weight-bold">
+        {status}
+        {children ? <span className={classes('extra')}>{status ? ' - ' : null}{children}</span> : null}
```

Comment:
I was able to create the component using the text `<p>test</p>` instead of a button. Is this intentional? Might be worth, as Daniel suggested, making it so that users just provide the information to display in the button and where to link to and we create the button automatically

>   <StatusMessage status="My heading" actions={[
>     { href: '#', text: 'Run' }
>     { href: '#', text: 'Jump' }
>     { href: '#', text: 'Sleep' }
>   ]}>
>     extra info
>   </StatusMessage>
> );

Reply:
I've changed the interface to expect an array of objects, as suggested

Reply:
I can confirm this is fixed.